src/components/users/views.tsx

```diff
@@ -101,3 +121,168 @@ export function UserPage(props: IUserPageProps): ReactElement {
     </>
   );
 }
+
+export function PasswordResetRequest(props: IPasswordResetFormProperties): ReactElement {
+  return <div className="govuk-grid-row">
+    <div className="govuk-grid-column-two-thirds">
+      <h1 className="govuk-heading-l">Password reset</h1>
+      <p className="govuk-body">
+        We will send you an activation link via email. If you do not receive an
+        email in the next 24 hours, your account may not exist in the system or
+        you could have setup authentication with Google or Microsoft Single
+        Sign-on.
+      </p>
+
+      {props.invalidEmail ? <div
```

Comment:
can we move the error-summary to the top of the page as documented in https://design-system.service.gov.uk/components/error-summary/